tasks.yaml

```diff
@@ -117,10 +119,7 @@ tasks:
     description: Configure existing uds-core/grafana package for group authz, build, deploy grafana
     actions:
       - cmd: |
-          sed -i '/^\s*sso:/,/^\s*-\s*name:/ {
```

Comment:
To reviewers: Changes in this file will be removed once https://github.com/defenseunicorns/uds-core/pull/1382 is integrated.